tests/unit/test-cmd/test.build.js

```diff
@@ -49,6 +49,25 @@ describe('build', () => {
     );
   });
 
+  it('configure a build', () => {
+    const packageCreator = sinon.spy(
+      () => ({extensionPath: 'extension/path'})
+    );
+    const fileFilter = {wantFile: () => true};
+    const createFileFilter = sinon.spy(() => fileFilter);
+    const params = {
+      sourceDir: '/src',
+      artifactsDir: 'artifacts',
+      ignoreFiles: ['**/*.log'],
+    };
+    return build(params, {packageCreator, createFileFilter}).then(() => {
+      // ensure sourceDir, artifactsDir, ignoreFiles is used
+      assert(createFileFilter.calledWithMatch(params));
+      // ensure packageCreator received correct fileFilter
+      assert(packageCreator.calledWithMatch({fileFilter}));
```

Comment:
This error would be easier to debug if it reported the incorrect params. I suggest this instead:
````js
assert.ok(packageCreator.called);
assert.deepEqual(packageCreator.firstCall.args[0], params);
````